docs/guides/universal-receiver-delegate/set-default-implementation.md

```diff
@@ -0,0 +1,110 @@
+---
+sidebar_label: 'Set the default Implementation'
+sidebar_position: 1
+---
+
+# Set the default Implementation
+
+:::note
+
+Users deploying their Universal Profiles using the guides that utilize **[lsp-factory](../universal-profile/create-profile.md)** or the **[Browser Extension](../browser-extension/create-a-universal-profile.md)** can skip this guide, as this contract is already deployed and set for their profiles.
+
+:::
+
+This guide will teach you how to deploy and set the default implementation of the **[Universal Receiver Delegate](../../standards/smart-contracts/lsp1-universal-receiver-delegate-up.md)** (URD) used by the Universal Profile. This contract will register the addresses of the **received assets and vaults** and will remove them on a balance equal to 0. This contract requires the [SETDATA Permission](../../standards/universal-profile/lsp6-key-manager.md#permissions) to interact with the profile through the KeyManager.
```

Comment:
Maybe we can add a link to LSP5 and LSP10 for **received assets** and **owned vaults**?.

```suggestion
This guide will teach you how to deploy and set the default implementation of the **[Universal Receiver Delegate](../../standards/smart-contracts/lsp1-universal-receiver-delegate-up.md)** (URD) used by the Universal Profile. This contract will register the addresses of the **received assets** and **owned vaults** and will remove them on a balance equal to 0. This contract requires the [SETDATA Permission](../../standards/universal-profile/lsp6-key-manager.md#permissions) to interact with the profile through the KeyManager.
```